one_time_payment_view.php

```diff
@@ -118,9 +97,34 @@ function gotoStripe() {
           } else {
             error.insertAfter(element);
           }
+        },
+          submitHandler: function(form) {
+		      form.submit();
```

Comment:
replace this with gotoStripe()?  Right now, you appear to be simply invoking the default behavior.